test/utils/on-and-off.js

```diff
@@ -12,7 +12,7 @@ const clean = require('../utils/clean')
 const os = require('os')
 
 const DaemonFactory = require('ipfsd-ctl')
-const df = DaemonFactory.create()
+const df = DaemonFactory.create({port: 0})
```

Comment:
No need for port. CLI tests are all run from Node.js. In fact this port option is doing nothing.